test/colorchanger/colorchanger_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Elixirlight.ColorchangerTest do
+  use ExUnit.Case, async: true
+
+  test "gets initial color" do
+    {:ok, color} = Elixirlight.Colorchanger.start_link
+    assert Elixirlight.Colorchanger.get(color) == "#FFFFFF"
```

Comment:
Maybe just make sure it equals the color of a new color that you create in this test?